src/pint/observatory/__init__.py

```diff
@@ -18,6 +19,8 @@
 from pint.pulsar_mjd import Time
 from pint.utils import interesting_lines
 
+# has the lead-second file been downloaded?
+_leapsecond_download = False
```

Comment:
Is this needed?

Reply:
I don't know.  I suspect not, but it's pretty harmless (in terms of runtime) so I left it in.